rfcs/0011_Arm_Ethos-U_Integration.md

```diff
@@ -0,0 +1,233 @@
+    Feature Name: Arm® Ethos™-U Integration
+    Start Date: 2020 May
+    RFC PR: https://github.com/apache/tvm-rfcs/pull/11
+    GitHub Issue: https://github.com/apache/tvm/issues/8482
+
+# Motivation
+
+Arm® Ethos™-U is a series of NPUs that will enable low-cost and highly efficient AI solutions for a wide range of embedded devices. This RFC introduces the port of Ethos-U into the uTVM compilation flow. The process of compilation relies on the multiple levels of abstraction in TVM and a variety of analysis and optimisation passes to produce c output. In the process of compilation, we rely on the many levels of TVM's IR (and the passes) to perform optimizations to create c-sources that can work with current microTVM deployments.
+
+## Scope:
```

Comment:
could you explain the intent behind this RFC, so readers know whether it's comprehensive or whether to expect follow-on RFCs?

Reply:
I think its explained : "The scope for this RFC is to add support for offloading to the Arm® Ethos™-U55 NPU. The initial machine learning framework that we use for testing this is TensorFlow Lite. Future RFCs and pull requests will address additional NPUs such as the Ethos™-U65, optimization to the compilation pipeline and other frameworks as the port evolves."